### DISCUSSION TITLE
New Mazda 3 - custom PIDs

### DISCUSSION THREAD

**Discussion by MasterPIC** [agronick/aa-torque#74] [Q&A]
Hi,

did you test wheel pressure PIDs on your Mazda 3?

An user reported these as working:

> Name ShortName ModeAndPID Equation Min Value Max Value Units Header scale minimumRefreshDelayMillis
> 
> Wheel 1 Pressure W1 Press 0x222a05 ((A*1373)/1000)/100 0 3.5 Bar 720 1 0
> Wheel 2 Pressure W2 Press 0x222a06 ((A*1373)/1000)/100 0 3.5 Bar 720 1 0
> Wheel 3 Pressure W3 Press 0x222a07 ((A*1373)/1000)/100 0 3.5 Bar 720 1 0
> Wheel 4 Pressure W4 Press 0x222a08 ((A*1373)/1000)/100 0 3.5 Bar 720 1 0

Could a too high polling rate make the sensors discharging? How to reduce polling rate?

As for fuel consumption could you share your Torque settings?

Thanks
Armando

**Comment by agronick** [comment]
They don't work for me. Maybe they don't work on gen 4. Polling won't change how often the signal is sent.
